hooks/tk-flame_actions.py

```diff
@@ -321,13 +321,19 @@ def _get_paths_from_published_files(self, sg_published_files):
 
             file_info = self.parent.shotgun.find_one(sg_type, filters=sg_filters, fields=sg_fields)
 
-            path = self.get_publish_path(file_info)
+            try:
+                path = self.get_publish_path(file_info)
+            except TankError:
```

Comment:
A comment explaining why an exception can be thrown would be nice for the next person who updates that code.